docs/dev-storm.md

```diff
@@ -30,6 +47,23 @@ This section shows how to run an existing Storm jar in a local Gearpump cluster.
    
    *Note that submission from UI is not supported yet*. 
 
+## Security support for Storm connectors
```

Comment:
Security support for Storm connectors

How about change the tile to xx limitations  xx?